plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/common/core/utils/StringUtils.java

```diff
@@ -192,4 +193,20 @@ public static String serialize(Map<String, String> map){
 		Collections.sort(out);
 		return org.apache.commons.lang.StringUtils.join(out.toArray(), ",");
 	}
+
+	/**
+	 * Remove all trailing <code>/</code> from a {@link String}.
+	 * @param value
+	 * @return the value without trailing <code>/</code>
+	 */
+	public static String removeTrailingSlashes(String value) {
+		if (value == null){
+			return null;
+		}
+		String newValue = value;
+		while(newValue.endsWith(SEPARATOR)) {
```

Comment:
Found this to work too: https://github.com/openshift/openshift-restclient-java/commit/93f1029fcdec4291b97d9857410549f188f97ff8#diff-84640426c91eb9d55ca08cc272d4a330R87   but im not picky